assets/training/model_management/tests/dev_requirements.txt

```diff
@@ -1,4 +1,5 @@
 # src
+pycocotools==2.0.4
```

Comment:
2.0.4 doesn't compile with Cython 3.0 & fails during pip install. 

We could do either of the following: 
1. Move pycocotools=2.0.4 to conda section to pull from conda-forge
2. Change to pycocotools==2.0.6 and keep continuing to use pip

Reply:
updated to 2.0.6 in pip dep